src/components/User/InfoForm.tsx

```diff
@@ -43,25 +47,14 @@ const ButtonBox = styled(Box)`
 `;
 
 interface Info {
-  identity_card: string;
-  name: string;
-  dob: string;
-  gender: string;
+  identity_card: string | undefined;
+  name: string | undefined;
+  dob: string | undefined;
+  gender: string | undefined;
```

Comment:
```suggestion
  identity_card?: string;
  name?: string;
  dob?: string;
  gender?: string;
```

Viết thế này cho ngắn, viết thế kia vẫn yêu cần phải thêm undefined khi tạo object